### PR TITLE
CMQ-2428: clarify responsibility for choosing correct tech stack

### PR DESCRIPTION
**Pull request by matthewmascord**
* Avoided use of the word architecture as it is already used to describe connection method
* Added in three places: what you need to send, connection method and public IP and port sections

### CODE REVIEW

File: source/connection-method/index.html.md.erb [hunk 1]
```diff
@@ -52,6 +55,17 @@ Only select this for requests when your users do not initiate an action and your
   </div>
 </details>
 
+### Choose the right technology stack
+
+Before building your application, make sure you select technologies that will allow
+you to send all header data required for your connection method. For example, some popular load balancer
+products do not pass on users’ public IPs or ports.
```
Comment (czeczotka) [inline source/connection-method/index.html.md.erb:62]: Perhaps `solutions` or `technologies` rather then `products`.

Comment (czeczotka) [reply source/connection-method/index.html.md.erb:62]: I wonder if we could turn this into a positive case: 
`For example, check if you selected load balancer can pass users' public IP addresses and ports.`

File: source/connection-method/index.html.md.erb [hunk 2]
```diff
@@ -52,6 +55,17 @@ Only select this for requests when your users do not initiate an action and your
   </div>
 </details>
 
+### Choose the right technology stack
+
+Before building your application, make sure you select technologies that will allow
+you to send all header data required for your connection method. For example, some popular load balancer
```
[the remark this answers is not in the document]
Comment (matthewmascord) [reply source/connection-method/index.html.md.erb]: Thanks, have re-worded.

File: source/connection-method/index.html.md.erb (hunk 1 above)
Comment (matthewmascord) [reply source/connection-method/index.html.md.erb]: Have removed the word product - weren't keen on `solutions` or `technologies`

File: source/connection-method/index.html.md.erb [hunk 3]
```diff
@@ -13,6 +13,9 @@ Your connection method depends on <a href="#connecting-to-hmrc">how your applica
 
 To find out what data you need to send, select the connection methods used by your application.
 
+It is your responsibility to [choose the right technology stack](#choose-the-right-technology-stack) to support the collection of all header data required for your connection method.
+Some popular ones do not.
```
[the remark this answers is not in the document]
Comment (matthewmascord) [reply source/connection-method/index.html.md.erb:17]: Discussed but decided to leave as separate sentence for emphasis.

**Review by czeczotka — APPROVED** [review]

**Review by czeczotka — APPROVED** [review]

**Pull request merged**